cmd/query/app/server.go

```diff
@@ -114,6 +119,7 @@ func (s *Server) Start() error {
 	// Start GRPC server concurrently
 	go func() {
 		s.svc.Logger.Info("Starting GRPC server", zap.Int("port", s.queryOptions.Port))
+
 		if err := s.grpcServer.Serve(grpcListener); err != nil {
```

Comment:
this is the same pattern as with the other two servers, but it works fine?

Reply:
Yes, grpc doesn't complain. I guess it has to do with how `grpc.Serve()` is implemented. Unlike with `http.Serve()`, grpc doesn't throw errors after `grpc.Stop()`